README.md

```diff
@@ -49,5 +49,5 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is inspired to **Cindy Shin in Behance** template CAMPANY STUFF_Conferance
+This project is inspired to **Cindy Shin in Behance** template CC-Global-Summit
```

Comment:
- [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly implement it

Reply:
Thank you for the feedback, happy to hear about it and I work on it